src/bin/ci/resources/papi_application.inc.conf.ctmpl

```diff
@@ -5,7 +5,7 @@ backend {
 }
 
 engine {
-  filesystems {
+  file
```

Comment:
Hmm I'm pretty sure this should be `filesystems` no ?

Reply:
yup! typo :(